standard/expressions.md

```diff
@@ -968,6 +968,16 @@ For a function member that includes a parameter array, if the function member is
   - for a fixed value parameter or a value parameter created by the expansion, an implicit conversion ([§10.2](conversions.md#102-implicit-conversions)) exists from the argument expression to the type of the corresponding parameter, or
   - for a `ref` or `out` parameter, the type of the argument expression is identical to the type of the corresponding parameter.
 
+In the case of a method group
+
+- If the group contains both instance and static members, instance members invoked without an instance receiver or context are not applicable.
+- If the group contains both instance and static members, static members invoked with an instance receiver are not applicable.
+- When there is no receiver, static members in a static context are applicable; otherwise, both static and instance members are applicable.
+- When the receiver is ambiguously an instance or type due to a color-color situation, both are applicable.
+- A static context, where an implicit `this` instance receiver cannot be used, includes the body of members where no `this` is defined, such as static members, as well as places where this cannot be used, such as field initializers and constructor initializers.
```

Comment:
With the above bullet removed, we don't need to define static context here - although we probably *do* need to define instance context. The term "static context" is only used elsewhere in the definition of the `nameof` operator. It probably makes sense to define "instance context" and "static context" somewhere, but it's not clear to me where that should be.

Reply:
Hmm... I've tried something else in the end, i.e. clarifying where an instance method with no receiver is valid.